internal/backends/collection.go

```diff
@@ -83,7 +83,7 @@ type QueryParams struct {
 	Filter        *types.Document
 	Sort          *SortField
 	Limit         int64  // if 0 no limit pushdown is applied
-	OnlyRecordIDs bool   // TODO https://github.com/FerretDB/FerretDB/issues/3490
+	OnlyRecordIDs bool   // ignored if collection is not capped collection
```

Comment:
It should not be ignored. The result might be nonsensical for now, but we should not add special case in backend implementations